json_annotation/lib/src/json_serializable.dart

```diff
@@ -65,10 +65,16 @@ class JsonKey {
   /// enclosing class.
```

Comment:
Also: bump the minor version of `json_annotation` in the pubspec to `0.2.3-dev`.

Add an entry to the changelog (under `0.2.3`) that includes this change.